apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,146 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  NotificationMarker,
+  NotificationMarkerProps,
```

Comment:
Want to confirm that this issue is just on my end: 
![image](https://user-images.githubusercontent.com/37936169/208146137-7c313e20-efa2-4b94-b904-e999ad0c46b3.png)

Reply:
This is not happening to me, but it might be alright. I too get similar errors sometimes but it occasionally goes away after restarting VSCode or re-running `yarn install`. Mayank also suggested to try running the `TypeScript: Restart TS Server` VSCode command to see if it helps, since Storybook is usually unreliable.

But for now, I think this error can be ignored.